1.hello-world/README.md

```diff
@@ -32,9 +32,9 @@ cd samples/1.hello-world
 In the `app.js` you'll find a simple `express` application, which exposes a few routes and handlers. First, let's take a look at the `stateUrl` at the top of the file: 
 
 ```js
-const stateUrl = `http://localhost:${daprPort}/v1.0/state`;
+const stateUrl = `http://localhost:${daprPort}/v1.0/state/${stateStoreName}`;
 ```
-When we use the Dapr CLI, it creates an environment variable for the Dapr port, which defaults to 3500. We'll be using this in step 3 when we POST messages to our system.
+When we use the Dapr CLI, it creates an environment variable for the Dapr port, which defaults to 3500. We'll be using this in step 3 when we POST messages to our system. The stateStoreName is the name of the state store being used which is configured in the yaml file under components.
```

Comment:
> The stateStoreName is the name of the state store being used which is configured in the yaml file under components.

stateStoreName should be in ticks `stateStoreName`.

This should indicate the exact location for stateStoreName within the yaml.  Ideally a snippet of a yaml with a highlight of the proper line.

Reply:
Added ticks to stateStoreName.
Added a yaml snippet and comment to point the line.